tests/read_only_trx_test.py

```diff
@@ -69,8 +79,20 @@
     Print ("producing nodes: %s, non-producing nodes: %d, topology: %s, delay between nodes launch(seconds): %d" % (pnodes, total_nodes-pnodes, topo, delay))
```

Comment:
This test does not handle `--clean-run` correctly, please model it off another working test like `nodeos_run_test.py`